src/main/java/seedu/address/model/person/Participant.java

```diff
@@ -47,4 +68,21 @@ public boolean isSamePerson(Person other) {
 
         return super.isSamePerson(other);
     }
+
+    @Override
+    public ToStringBuilder toStringBuilder() {
+        return super.toStringBuilder()
+                .add("group", group);
+    }
+
+    @Override
+    public String toCsvLine() {
+        return super.toCsvLine() + "," + group;
```

Comment:
I think we dont need to save group number into csv as it is not an information that sponsors want or need to know.